packages/core-plugin-api/src/apis/definitions/auth.ts

```diff
@@ -152,6 +152,26 @@ export type OpenIdConnectApi = {
   getIdToken(options?: AuthRequestOptions): Promise<string>;
 };
 
+/**
+ * This API provides access to Pinniped credentials. It lets you request Supervisor and Cluster Scoped ID tokens,
+ * which can be passed to backend services to exchange it for mTLS x509 client certs.
```

Comment:
we changed it so it doesn't let you request supervisor ID tokens!
```suggestion
 * This API provides access to Pinniped credentials. It lets you request Cluster Scoped ID tokens,
 * which can be passed to a suitably-configured Pinniped Concierge in exchange for x509 client cert bundles.
```